idc.Makefile

```diff
@@ -48,6 +48,30 @@ snapshot-image:
 	$(MAKE) docker-compose.yml
 	docker-compose up -d
 
+.PHONY: reset
+.SILENT: reset
+reset: warning-destroy-state destroy-state
+	@echo "Removing vendored modules"
+	-rm -rf codebase/vendor
```

Comment:
If the intent is to blow away modules & dependencies, and allow composer to re-populate then the following directories should also be emptied:
```
codebase/web/core/
codebase/web/modules/contrib/
codebase/web/themes/contrib/
```